File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UnitLibraryService.Models/get-unit-alias_item.ts


import { IGetUnitAlias_Item } from './iget-unit-alias_item';

export class GetUnitAlias_Item implements IGetUnitAlias_Item {
  
	public UnitAliasID: number = 0;
	public UnitID: number = 0;
	public UnitAlias: string = "";
	public LanguageID?: number;
	public IsDefault: boolean = false;
	public IsActive: boolean = false;
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
  
}

File: GTTS/INL.UnitLibraryService.Models/GetUnitAlias_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.UnitLibraryService.Models
{
    public class GetUnitAlias_Item : IGetUnitAlias_Item
    {
        public int UnitAliasID { get; set; }
        public long UnitID { get; set; }
        public string UnitAlias { get; set; }
        public int? LanguageID { get; set; }
        public bool IsDefault { get; set; }
        public bool IsActive { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}
